easybuild/easyconfigs/s/STELLA/STELLA-4a5f9c5-CrayGNU-20.06-crclim-double.eb

```diff
@@ -0,0 +1,41 @@
+# omlins, kraushm, jg, havogt (cscs)
+easyblock = 'CMakeMake'
+
+name = 'STELLA'
+version = '4a5f9c5'
+versionsuffix = '-crclim-double'
+
+homepage = "https://github.com/C2SM-RCM/stella (-b crclim)"
+description = 'STELLA'
+
+toolchain = {'name': 'CrayGNU', 'version': '20.06'}
+toolchainopts = {'verbose': False}
+
+# c2sm repo is private (+ no official release)
+sources = [{'filename': '/apps/common/UES/easybuild/sources/s/%(name)s/%(namelower)s_%(version)s.tar.gz'}]
+
+builddependencies = [
+    ('CMake', '3.12.0', '', True),  # 3.14.5 #3.12.0
+    ('cudatoolkit/10.1.105_3.27-7.0.1.1_4.1__ga311ce7', EXTERNAL_MODULE),
+    # ('cudatoolkit/10.2.89_3.28-7.0.1.1_2.1__g88d3d59', EXTERNAL_MODULE),
+    ('gcc/8.3.0', EXTERNAL_MODULE),  # Ensure that this is loaded. Fails if not.
```

Comment:
Is this still needed? it should be the default (or loaded by the CrayGNU toolchain)?

Reply:
I added this to ensure that the right gcc version is loaded. As it says in the comment: it ensures that it is loaded, else it fails. I will remove this again once everything works if you think I should or I improve the comment.